handbook/product/goals.md

```diff
@@ -85,6 +85,15 @@ _Updated 2020-09-18_
 - **Metrics:**
    - Increase visits
    - Increase creation of and usage of extensions
+   
+### Create baseline for public (and eventually private) code on Sourcegraph Cloud
+
+- **Owner:** Quinn
+- **Timeline:** ~2 months
+- **Status:** In progress
+- **Outcome:** Any user can add public code from GitHub.com, GitLab.com, and Bitbucket Cloud.[*N<sub>9</sub>*][N9]
```

Comment:
The reference numbers are for metrics we don't want to be public (links in a Google Doc). In this case it's just a link to another section of the docs, so we can be clear about what this link is :) 

```suggestion
- **Outcome:** Any user can add public code from GitHub.com, GitLab.com, and Bitbucket Cloud. Support accomplishing [Cloud milestones 1 and 2](https://about.sourcegraph.com/handbook/engineering/cloud#milestones).
```